src/server.c

```diff
@@ -4043,6 +4045,10 @@ int processCommand(client *c) {
         call(c, flags);
         if (listLength(server.ready_keys) && !isInsideYieldingLongCommand()) handleClientsBlockedOnKeys();
     }
+
+    /* Now that c->slot has been parsed, and command has been executed,
+     * accumulate the buffered network bytes-in. */
+    clusterSlotStatsAddNetworkBytesIn(c);
```

Comment:
We can maybe move this under `afterCommand`.

Reply:
I think we would want to accumulate it on unblocking cases, so `afterCommand` makes sense.